ci/test_python_cudf.sh

```diff
@@ -2,6 +2,8 @@
 # Copyright (c) 2022-2025, NVIDIA CORPORATION.
 
 # Support invoking test_python_cudf.sh outside the script directory
+# TODO: Fix shellcheck error
+# shellcheck disable=SC2164
```

Comment:
```suggestion
```

Reply:
pre-commit was failing locally for me without this change. We'll see if pre-commit.ci says the same